Source/CesiumRuntime/Private/CesiumSubLevelSwitcherComponent.cpp

```diff
@@ -303,7 +294,8 @@ void UCesiumSubLevelSwitcherComponent::_updateSubLevelStateGame() {
 
       // Double-check that we're not actively trying to unload this level
       // already. If we are, wait longer.
-      if (IsValid(pStreaming) && pStreaming->ShouldBeLoaded()) {
+      if ((IsValid(pStreaming) && pStreaming->ShouldBeLoaded()) ||
+          this->_pTarget.Get()->GetWorldAsset().IsNull()) {
```

Comment:
Just checking, what does it mean if the target level's world asset is null? Do we want the current sublevel to have a null world asset?

Reply:
It means that we have an ALevelInstance that doesn't actually have any level associated with it at all. We can still switch to such a sub-level, it just doesn't have any content. There's a comment on a similar GetWorldAsset check above.